requirements.txt

```diff
@@ -1,23 +1,171 @@
-boto3
-kombu==4.6.3 # CeleryExecutor issues with 1.10.2 supposedly fixed in 1.10.5 airflow, but still observed issues on 1.10.7
```

Comment:
this needs to be pinned

Reply:
celery 4.4.2 is what prod currently uses.

Reply:
This was an issue on my local development machine, `kombu==4.6.3` is incompatible with `celery==4.4` when running `pip-compile` on python2. I used a `python3.7` virtualenv which compiles this fine.